gnomad/sample_qc/relatedness.py

```diff
@@ -987,24 +985,29 @@ def maximal_independent_set_keep_samples(
     return related_samples_to_drop_ht
 
 
-def filter_mt_to_trios(mt: hl.MatrixTable, fam_ht: hl.Table) -> hl.MatrixTable:
+def filter_mt_to_trios(
+    t: Union[hl.vds, hl.matrixtable], fam_ht: hl.Table
```

Comment:
i believe "t" is typically used for an MT or HT, but we use "mtds" when it's either a MT or a VDS

Reply:
Good point